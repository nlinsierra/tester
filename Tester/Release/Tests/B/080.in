2H 9D 6D TS TD 7S 7H 4C 9H KC 6H 8D 3D 6C 9C
