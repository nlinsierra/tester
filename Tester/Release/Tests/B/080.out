37
2C 3C 5C 7C 8C TC JC QC AC 2D 4D 5D 7D JD QD KD AD 3H 4H 5H 8H TH JH QH KH AH 2S 3S 4S 5S 6S 8S 9S JS QS KS AS
